src/open_inwoner/plans/models.py

```diff
@@ -115,6 +116,15 @@ class Plan(models.Model):
         blank=True,
         help_text=_("How do you intend to achieve this goal?"),
     )
+    note = ckeditor_fields.RichTextField(
+        max_length=1000,
+        default=_(""),
```

Comment:
No default

Reply:
Yeah, adding translation to that was silly... (happened because I had something in there first). But I do see we have multiple models in the project that have `default="",` but i will just completely remove the default if that works.